kinoml/ml/torch_models.py

```diff
@@ -22,11 +22,16 @@ def estimate_input_shape(input_sample):
         If your ``.forward()`` method takes something else
         than a Tensor, please adapt this method accordingly.
         """
-        if type(input_sample) == list:
-            return input_sample[0].shape[1]
-        else:
-            return input_sample.shape[1]
-
+        if len(input_sample)==1: # ligand-only
```

Comment:
I'm a bit confused why `input_sample` has such complex structure that requires all the special cases here.
Can we document the convention we're using for storing data in `input_sample` a bit more thoroughly, or point to where that is documented if it's documented already?

Reply:
Will those if statements catch all input_sample formats that may show up in future?